src/Styleguide/Components/MarketInsights.tsx

```diff
@@ -100,7 +98,15 @@ export class MarketInsights extends React.Component<MarketInsightsProps> {
       return null
     }
 
-    const Container = this.props.border ? BorderBox : Box
+    let Container
+
+    if (this.props.Container) {
```

Comment:
interesting approach to pass this around...makes sense from the layout/logic perspective